resources/marketplace/extension-page.md

```diff
@@ -18,6 +20,7 @@ You can use the AWS Toolkit for Visual Studio Code to:
   * List available AWS CloudFormation stacks that contain AWS serverless applications.
   * List available AWS Lambda functions.
   * Invoke available Lambda functions.
+  * ![AWS Explorer](./aws-explorer.png)
```

Comment:
Thinking we should move this image out of the bullet points. Try either line 15 or 26 (line 15 seems more in-line with what we're doing with the other images)